sentry_sdk/integrations/asgi.py

```diff
@@ -175,7 +176,7 @@ async def _run_app(self, scope, callback):
 
                     with hub.start_transaction(
                         transaction, custom_sampling_context={"asgi_scope": scope}
-                    ):
+                    ), start_profiling(transaction, hub):
```

Comment:
I'm not too familiar with how ASGI works but I tried to run a Django app in ASGI using gunicorn + uvicorn and I noticed that this happens on the main thread and the request is handled on another thread. Ideally, we can detect the thread handling the request so we can show it in the UI by default. Right now, it's just showing the main thread which isn't very helpful.